package-lock.json

```diff
@@ -1,6416 +1,26 @@
 {
   "name": "play",
-  "lockfileVersion": 2,
   "requires": true,
-  "packages": {
```

Comment:
is it possible to revert this change? Would prefer to keep the existing package-lock :)

Reply:
upss, sorry! reverted back to the original package-lock :)